src/BlazorWebView/src/WindowsForms/BlazorWebView.cs

```diff
@@ -175,17 +175,17 @@ private void HandleRootComponentsCollectionChanged(object sender, NotifyCollecti
         /// <inheritdoc />
         protected override void Dispose(bool disposing)
         {
-            base.Dispose(disposing);
-
             if (disposing)
             {
-                _webviewManager?.Dispose();
-                _webview?.Dispose();
+				// Dispose this component's contents before calling base.Dispose() because that will dispose the WebView2 control, likely
+				// preventing user-written disposal logic from working (because it might try to use Blazor stuff, which wouldn't work anymore).
+				_webviewManager?.Dispose();
             }
-        }
+			base.Dispose(disposing);
+		}
 
-        /// <inheritdoc />
-        protected override ControlCollection CreateControlsInstance()
+		/// <inheritdoc />
+		protected override ControlCollection CreateControlsInstance()
```

Comment:
```suggestion
        /// <inheritdoc />
        protected override ControlCollection CreateControlsInstance()
```

Looks like spaces got converted to tabs.

Reply:
A bot will clean this up later :)